network-api/networkapi/mozfest/templates/partials/primary_mozfest_nav.html

```diff
@@ -0,0 +1,70 @@
+{% load primary_active_nav %}
+
+<div id="primary-nav-container">
+  <div class="wrapper-burger">
+    <div class="menu-container">
+      <div class="narrow-screen-menu hidden d-lg-none">
+        <div class="narrow-screen-menu-background">
+          <div class="narrow-screen-menu-container">
+            <nav class="container">
+              <div class="row">
+                <div class="col">
+                  <div class="nav-links pt-3">
+                    <div><a class="{% primary_active_nav request '/' %}" href="/">Home</a></div>
+                    <div><a class="{% primary_active_nav request '/about' %}" href="/about">About</a></div>
+                    <div><a class="{% primary_active_nav request '/spaces' %}" href="/spaces">Spaces</a></div>
+                    <div><a class="{% primary_active_nav request '/team' %}" href="/team">Team</a></div>
+                    <div><a class="{% primary_active_nav request '/tickets' %}" href="/tickets">tickets</a></div>
+                    {% if HEROKU_APP_NAME %}
+                    <div><a href="/help">DEV HELP</a></div>
+                    {% endif %}
+                  </div>
+                </div>
+              </div>
+            </nav>
+          </div>
+        </div>
+      </div>
+      <nav class="container">
+        <div class="row">
+          <div class="col">
+            <div class="d-flex flex-row justify-content-between">
+              <div id="primary-nav-links">
+                <div class="d-flex align-items-center flex-wrap">
+                {% if page.zen_nav == True %}
```

Comment:
there's still some cruft like this that needs to get trimmed out because mozfest has no zen/narrow etc, to be confirmed in the comments